src/pp.js

```diff
@@ -360,7 +367,13 @@ function print(options, doc) {
 
                 pos = 0;
               } else {
-                out.push("\n" + _repeatIndent(ind, indentStr));
+                let indentLine;
+                if (ind in indentCache) {
+                  indentLine = indentCache[ind];
+                } else {
+                  indentCache[ind] = indentLine = _repeatIndent(ind, indentStr);
+                }
+                out.push("\n" + indentLine);
```

Comment:
I just realised - it *is* negligible, because it would be called only once for each *level* of indentation.  I will proceed to send the cache object to the repeatIndent function.